org.eclipse.dawnsci.remotedataset.client/src/org/eclipse/dawnsci/remotedataset/client/RemoteDataset.java

```diff
@@ -234,6 +234,10 @@ private void createInfo() throws Exception {
 		
 		if (this.name == null) this.name  = info.get(0);
 		this.shape = toIntArray(info.get(1));
+		if (this.shape.length == 1 && shape[0] == -1) {
+			System.err.println();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)